superset/sql_lab.py

```diff
@@ -171,6 +171,7 @@ def handle_error(msg):
     # Limit enforced only for retrieving the data, not for the CTA queries.
     superset_query = SupersetQuery(rendered_query)
     executed_sql = superset_query.stripped()
+    SQL_MAX_ROWS = int(app.config.get('SQL_MAX_ROW', None))
```

Comment:
`None` is implicit on the get method and will make `int` raise
```
In [1]: int(None)
TypeError: int() argument must be a string, a bytes-like object or a number, not 'NoneType'
```

Reply:
I'll send out the fir for this.